pages/examples/animations/fade.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+
+import GlobalModalManager from '@system/modals/GlobalModalManager';
```

Comment:
This needs to follow alphabetization and the structure that is in our other files, please make this change when you get a chance.